src/discordas.ts

```diff
@@ -176,6 +176,11 @@ async function run() {
     const roomhandler = discordbot.RoomHandler;
     const eventProcessor = discordbot.MxEventProcessor;
 
+    // tslint:disable-next-line:no-any
+    appservice.expressAppInstance.get("/health", (_, res: any) => {
+        res.status(201).send("");
```

Comment:
Is there a particular reason to return [201](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/201)?

Reply:
Only that the Slack bridge does it this way... `Created` does not make any sense at all.
Lets change it to 200.

Reply:
If there was no reason to use 201, surely 204 No Content is better?

Reply:
I think 204 is for omitting even though there may be content.
It expects the server to send an `ETag` header to indicate that the content didn't change.
https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/204

Responding 200 without content actually means "There is no content".

Reply:
I think that's saying you don't need to send a Cache-Control header for a 204 to be cacheable.